src/test/java/com/wikia/webdriver/testcases/communitypagetests/SpecialCommunityPageTests.java

```diff
@@ -0,0 +1,58 @@
+package com.wikia.webdriver.testcases.communitypagetests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.communitypage.SpecialCommunity;
+
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+
+@Test(groups = "CommunityPageTests")
+public class SpecialCommunityPageTests extends NewTestTemplate {
+
+  @Execute(onWikia = "muppet", asUser = User.ANONYMOUS, mockAds = "true")
+  public void verifyAnonEditingForbiddenFlow() {
+    SpecialCommunity page = new SpecialCommunity();
+    // click the first link from any card
+    WebElement link = page.getLinkFromCards();
+    String url = getEditUrl(link.getText(), "veaction=edit");
+    link.click();
+    // should be at register page
```

Comment:
+1 Can we verify we are on register page. Also do we want to cover case when new user is joining community?